test/extended/core.sh

```diff
@@ -97,4 +97,4 @@ fi
 echo "[INFO] Running extended tests"
 
 # Run the tests
-TMPDIR=${BASETMPDIR} ginkgo -progress -stream -v -p "-skip=${SKIP}" "$@" ${OS_OUTPUT_BINPATH}/extended.test
+TMPDIR=${BASETMPDIR} ginkgo -progress -stream -v "-skip=${SKIP}" "$@" ${OS_OUTPUT_BINPATH}/extended.test
```

Comment:
Correct, for now.  When I split the packages up I will run upstream non parallel, then us parallel